pom.xml

```diff
@@ -16,8 +16,21 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
+    <license.licenseName>MIT</license.licenseName>
   </properties>
 
+  <developers>
+    <developer>
+      <name>Eclipse Trace Compass Community</name>
+      <url>eclipse.dev/tracecompass</url>
```

Comment:
Isn't it `organizationUrl` instead of `url`? Maybe use the https://github.com/eclipse-tracecompass as organization url.

Also add
```xml
   <organization>Eclipse Trace Compass</organization>
```